extension/src/popup/views/Loading.tsx

```diff
@@ -8,8 +18,39 @@ const LoadingEl = styled.div`
   align-items: center;
 `;
 
+const LoadingSpinnerEl = styled.div`
+  display: inline-block;
+  height: 5rem;
+  width: 5rem;
+
+  &:after {
+    content: " ";
+    display: block;
+    width: 2rem;
+    height: 2rem;
+    margin: 0.5rem;
+    border-radius: 50%;
+    border: 0.5rem solid ${COLOR_PALETTE.primary};
+    border-color: ${COLOR_PALETTE.primary} transparent ${COLOR_PALETTE.primary}
+      transparent;
+    animation: lds-dual-ring 1.2s linear infinite;
+  }
+
+  @keyframes lds-dual-ring {
+    0% {
+      transform: rotate(0deg);
+    }
+    100% {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
this is an open source css loading spinner. This is placeholder until design comes up with something custom to implement

Reply:
Maybe nice to include a link to the source of it, in that case